castWebApi.js

```diff
@@ -7,7 +7,7 @@ const url = require('url');
 const debug = require('debug')('cast-web-api');
 const args = require('minimist')(process.argv.slice(2));
 const fetch = require('node-fetch');
-var hostname = '127.0.0.1';
```

Comment:
I also changed this to 0.0.0.0 so the node server would listen on all ip's by default.

Reply:
Listening on one ip by default is a security precaution. Experienced the weirdest things on the forum already, where people forwarded ports out, connected to VPN (...) . Next release picks one interface ip by default. If a user wants it to listen on all ports, they can change it to 0.0.0.0 manually and are probably aware of what they're doing.